client/index.html

```diff
@@ -261,14 +267,14 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
-									<% themes.forEach(function(themeName) { %>
-										<option value="<%= themeName %>">
-											<%= themeName.charAt(0).toUpperCase() + themeName.slice(1) %>
+									{{#each themes}}
+										<option value="{{this}}">
+											{{firstLetterUppercase this}}
```

Comment:
I think that we don't want to remove helpers just for the sake of removing helpers. This is the way you are meant to do handlebars, and we do it for loads of stuff in the client.

I'm happy to call it "capitalize" but I thought that wasn't obvious enough because it isn't capitalising the whole thing, just the first letter. But if you think it's self explanatory enough, I guess.